application/app/src/test/java/uk/nhs/digital/uec/api/controller/FuzzySearchControllerTest.java

```diff
@@ -92,7 +93,7 @@ public void getServicesByFuzzySearchTestSucc() throws ValidationException {
     verify(mockValidationService, times(1)).validateSearchCriteria(searchCriteria);
     verify(mockValidationService, times(1)).validateMinSearchCriteriaLength(searchCriteria);
     verify(mockFuzzyServiceSearchService, times(1))
-        .retrieveServicesByFuzzySearch(SEARCH_POSTCODE, searchCriteria);
+        .retrieveServicesByFuzzySearch(SEARCH_POSTCODE, searchCriteria, null);
```

Comment:
here to

Reply:
These changes were already merged, this branch was rebased from master